# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/gscanbus/multilinestring.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN gscanbus-0.7.1/gscanbus.c gscanbus-0.7.1-new/gscanbus.c
--- gscanbus-0.7.1/gscanbus.c	2001-07-11 12:53:51.000000000 +0200
+++ gscanbus-0.7.1-new/gscanbus.c	2005-03-19 00:53:33.303914611 +0100
@@ -109,13 +109,13 @@
 
 static GdkPixmap *pixmap = NULL;
 
-const char not_compatible[] = "\
-This libraw1394 does not work with your version of Linux. You need a different
-version that matches your kernel (see kernel help text for the raw1394 option to
-find out which is the correct version).\n";
+const char not_compatible[] = 
+"This libraw1394 does not work with your version of Linux. You need a different\
+version that matches your kernel (see kernel help text for the raw1394 option \
+to find out which is the correct version).\n";
 
-const char not_loaded[] = "\
-This probably means that you don't have raw1394 support in the kernel or that
+const char not_loaded[] = 
+"This probably means that you don't have raw1394 support in the kernel or that \
 you haven't loaded the raw1394 module.\n";
 
 /*---------------------------------------------------------------------------
